package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@easyops-cn/docusaurus-search-local",
-  "version": "0.16.3",
+  "version": "0.16.4",
```

Comment:
Releases should only be published after merged into master branch (and for repo owners only).

Reply:
> Releases should only be published after merged into master branch (and for repo owners only).

done~